cmd/ocm-backplane/login/login.go

```diff
@@ -421,9 +414,27 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 		return err
 	}
 
+	logger.Debugln("Checking for other backplane sessions...")
+	cfg, _ := BuildRestConfig(bpAPIClusterURL, accessToken, proxyURL)
+	login.CheckForOtherSessions(cfg)
+
 	return nil
 }
 
+// BuildRestConfig takes a host, token and optional proxy URL and generates a rest config
+func BuildRestConfig(host string, token *string, proxyURL string) (*rest.Config, error) {
```

Comment:
I abstracted this functionality here since it would otherwise `doLogin` again if I reused the `GetRestConfig` or `GetRestConfigWithConn` functions, so that I could just build the rest config directly (because apparently it's not possible to go from an `api.Config` to the `*rest.Config` :/